### PR TITLE
Update reboot script to handle ssuboard systems

### PR DESCRIPTION
**Pull request by dangerware**
This reboot script will now handle ssuboard's lamassu-machine and lamassu-browser reboot through `supervirsorctl`.
Also added a readability improvement, regarding date handling, that was missing from #267.

### CODE REVIEW

**Comment by nurotyx** [comment]
[reboot update.tar for ssuboards](https://trello.com/c/Sr9zFj3U/153-reboot-updatetar-for-ssuboards)

**Comment by RafaelTaranto** [comment]
@dangerware Can you passthrough the previous value on the promise here? I'll add comments on the PR on how to do this.

File: lib/trader.js [hunk 1]
```diff
@@ -64,8 +64,17 @@ Trader.prototype.syncLogs = function syncLogs () {
     .then(data => data.body)
   // Delete log files that are two or more days old
     .then(() => {
```
Comment (RafaelTaranto) [inline lib/trader.js]: We need to pass data.body to logs.queryNewestLogs, name the param here.

```
.then((it) => {
```

File: lib/trader.js [hunk 2]
```diff
@@ -64,8 +64,17 @@ Trader.prototype.syncLogs = function syncLogs () {
...
+        let date = new Date()
+
+	// Notice that `setDate()` can take negative values. So if you'd take
+	// -2 days on the 1st of April you'd get the 30th of March. Several
+	// examples can be seen at: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate#Using_setDate()
+        date.setDate(date.getDate() - 2)
+	return date
+      })()
+
+      return logs.removeLogFiles(twoDaysAgo)
```
Comment (RafaelTaranto) [inline lib/trader.js]: now return the param, that way this function runs without interfering with queryNewestLogs

```
return it
```

File: lib/trader.js (hunk 1 above)
Comment (RafaelTaranto) [reply lib/trader.js]: Note that you can also refactor this delete stuff to a separate function and call it outside this promise chain, just make sure to add error handling to it.

File: lib/trader.js [hunk 3]
```diff
@@ -63,9 +63,18 @@ Trader.prototype.syncLogs = function syncLogs () {
...
+        let date = new Date()
+
+	// Notice that `setDate()` can take negative values. So if you'd take
+	// -2 days on the 1st of April you'd get the 30th of March. Several
+	// examples can be seen at: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate#Using_setDate()
+        date.setDate(date.getDate() - 2)
+	return date
+      })()
+
+      return logs.removeLogFiles(twoDaysAgo).then(it)
```
Comment (RafaelTaranto) [inline lib/trader.js]: `then()` expects a function as argument. This should be `then(() => it)`

Comment (joshmh) [reply lib/trader.js]: Thanks for the review, Rafi!

Comment (RafaelTaranto) [reply lib/trader.js]: It was mainly regarding the bug, though. Didn't look deep into the PR.

**Pull request merged**